cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/prometheus/PrometheusMetricSampler.java

```diff
@@ -52,6 +54,7 @@
 public class PrometheusMetricSampler extends AbstractMetricSampler {
     // Config name visible to tests
     static final String PROMETHEUS_SERVER_ENDPOINT_CONFIG = "prometheus.server.endpoint";
+    static final String PROMETHEUS_SERVER_ENDPOINT_BASIC_AUTH_CONFIG = "prometheus.server.endpoint.auth.basic";
```

Comment:
Whenever you are adding any configuration, please also add it to the documentation like configurations.md